analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -535,18 +557,25 @@ private static void VerifyClassMainVB(string members)
 $@"Public Class Sample
     {members}
 End Class";
-        VerifyCode<TestSERunnerVB>(code, ProjectType.Product, ImmutableArray<ParseOptions>.Empty, null, MainScopeAssignmentRuleCheck.SMain, BinaryRuleCheck.SBinary);
+        VerifyCode<TestSERunnerVB>(code, ProjectType.Product, ImmutableArray<ParseOptions>.Empty, null, OutputKind.DynamicallyLinkedLibrary, MainScopeAssignmentRuleCheck.SMain, BinaryRuleCheck.SBinary);
     }
 
-    private static void VerifyCode<TRunner>(string code, ProjectType projectType, ImmutableArray<ParseOptions> parseOptions, string sonarProjectConfigPath, params DiagnosticDescriptor[] onlyRules)
+    private static void VerifyCode<TRunner>(string code,
+                                            ProjectType projectType,
+                                            ImmutableArray<ParseOptions> parseOptions,
+                                            string sonarProjectConfigPath,
+                                            OutputKind kind,
+                                            params DiagnosticDescriptor[] onlyRules)
         where TRunner : SymbolicExecutionRunnerBase, new() =>
-        new VerifierBuilder<TRunner>()
-            .AddReferences(TestHelper.ProjectTypeReference(projectType))
-            .AddSnippet(code)
-            .WithSonarProjectConfigPath(sonarProjectConfigPath)
-            .WithOptions(parseOptions)
-            .WithOnlyDiagnostics(onlyRules)
-            .Verify();
+            new VerifierBuilder<TRunner>()
```

Comment:
If you put the `where` at the end of previous line, you will not needed to make this double tabbed body indent.

Just the fact that you can place the `where` and `=>` on 4 different places makes the C# syntax unpredictable and therefore hard to read